pkg/qiniu/qnPresubmit.go

```diff
@@ -132,3 +138,7 @@ func (a *Artifacts) CreateChangedProfile() *os.File {
 
 	return p
 }
+
+func (a *ProfileArtifacts) GetChangedProfileName() string {
```

Comment:
suggest to add comments for exported method

Reply:
added~